src/ProjectOrigin.WalletSystem.IntegrationTests/FlowTests/TransferTests.cs

```diff
@@ -165,4 +164,63 @@ private async Task WaitForCertCount(Guid certId, int number)
         }
         slicesFound.Should().Be(number, "correct number of slices should be found");
     }
+
+    [Theory]
+    [InlineData(250, 150)] // transfer part of slice
```

Comment:
I believe there is aproperty on the InlineData-attribute called `TestDisplayName `. You can use that instead of the comments.

Reply:
Can you find a reference for this with InlineData, I cant find it.

Reply:
I cannot find it either. 
I saw this https://github.com/xunit/xunit/discussions/2532, but failed to see that it will be part of v3 of xunit. It is 1 year old, but v3 has not been released yet.